e2e/integration-tests/integration_test.go

```diff
@@ -5,19 +5,32 @@ import (
 	"fmt"
 	"os"
 	"os/exec"
+	"path"
+	"time"
 
 	g "github.com/onsi/ginkgo/v2"
 	o "github.com/onsi/gomega"
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 )
 
-var _ = g.Describe("NetObserv CLI e2e integration test suite", g.Serial, func() {
+var (
+	StartupDate = time.Now().Format("20060102-150405")
+)
+
+var _ = g.Describe("NetObserv CLI e2e integration test suite", g.Ordered, func() {
 	cliNS := "netobserv-cli"
 
+	g.BeforeAll(func() {
+		createErr := os.MkdirAll(path.Join("output"), 0700)
```

Comment:
in CI, `$ARTIFACT_DIR` is exposed where each step artifacts can be stored, could you write to that dir?

Reply:
Sure !